contracts/storage/OptionsVaultStorage.sol

```diff
@@ -101,22 +88,21 @@ contract OptionsVaultStorageV3 {
     uint256 internal _totalPending;
 
     /// @notice Stores the user's pending deposit for the round
-    mapping(address => VaultDeposit.DepositReceipt) public depositReceipts;
+    mapping(address => Vault.DepositReceipt) public depositReceipts;
 
     /// @notice On every round's close, the pricePerShare value of an rTHETA token is stored
     /// This is used to determine the number of shares to be returned
     /// to a user with their DepositReceipt.depositAmount
     mapping(uint16 => uint256) public roundPricePerShare;
+
+    /// @notice Stores pending user withdrawals
+    mapping(address => Vault.Withdrawal) public withdrawals;
 }
 
 // We are following Compound's method of upgrading new contract implementations
 // When we need to add new storage variables, we create a new version of OptionsVaultStorage
 // e.g. OptionsVaultStorageV<versionNumber>, so finally it would look like
 // contract OptionsVaultStorage is OptionsVaultStorageV1, OptionsVaultStorageV2
-contract OptionsVaultStorage is
-    OptionsVaultStorageV1,
-    OptionsVaultStorageV2,
-    OptionsVaultStorageV3
```

Comment:
Decided to introduce incompatibility with Ribbon v1, we're going to do a migration of funds from v1 contract to v2 contract